packages/store/test/FieldLayout.t.sol

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { Test, console } from "forge-std/Test.sol";
+import { GasReporter } from "@latticexyz/gas-report/src/GasReporter.sol";
+import { FieldLayout, FieldLayoutLib } from "../src/FieldLayout.sol";
+import { FieldLayoutEncodeHelper } from "./FieldLayoutEncodeHelper.sol";
+
+// TODO add tests for all lengths
```

Comment:
it's meant in the same sense as `// TODO add tests for all schema types` in `Schema.t.sol`
we could fuzz/autogenerate tests for all the length configurations (or at least a lot more combinations), rather than testing a few edge cases

afaik we don't have an issue, made https://github.com/latticexyz/mud/issues/1453